src/simulation/LoadGenerator.cpp

```diff
@@ -630,14 +630,14 @@ LoadGenerator::TxInfo::execute(Application& app, bool isCreate,
     msg.transaction() = txf->getEnvelope();
     txm.mTxnBytes.Mark(xdr::xdr_argpack_size(msg));
 
-    auto status = app.getHerder().recvTransaction(txf);
-    if (status != TransactionQueue::AddResult::ADD_STATUS_PENDING)
+    auto result = app.getHerder().recvTransaction(txf);
```

Comment:
Nitpick: why did you rename this from `status` to `result`? It is antagonistic to your other change of renaming `AddResult` to `AddStatus`.